env_bootstrap/download.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh
+
+echo "hit download script."
+
+ROOT=$(pwd)
+echo "check root: " $ROOT
+
+AUDIO_DIR="$ROOT/raw_data"
+
+# if [ ! -d raw_data ]
```

Comment:
did you mean to comment this out? was the data download not succeeding? (note: it's slow)

Reply:
No, that needs to be brought back in. Will update